src/bin/trace/tracing_chrome.rs

```diff
@@ -0,0 +1,624 @@
+//! This file is taken unmodified from the following link, except for file attributes and
+//! `extern crate` at the top. It is licensed under MIT.
```

Comment:
The code is this repo is otherwise under MIT OR Apache2. I don't know if we have to do anything to license one file differently... I will have to [ask about this](https://rust-lang.zulipchat.com/#narrow/channel/392734-council/topic/Vendoring.20an.20MIT-licensed.20file/with/526194636).